src/components/MultiComboBox/useListbox.tsx

```diff
@@ -0,0 +1,316 @@
+    }
+    return undefined
+  })()
+
+  const theme = useTheme()
+  const { portalRoot } = usePortal()
+  const listboxRef = useRef<HTMLDivElement>(null)
+
+  const renderListbox = () => {
+    return createPortal(
+      <Container
+        {...dropdownStyle}
+        themes={theme}
+        id={listboxId}
+        className={isExpanded ? undefined : 'hidden'}
+        ref={listboxRef}
+        role="listbox"
+      >
+        {options.map((option, i) => {
+          const activeClass = activeOptionIndex === i ? 'active' : undefined
+          const { label, disabled, isAdding } = option
+          if (isAdding) {
+            return (
+              <AddButton
+                key={`add-${label}`}
+                themes={theme}
+                onClick={() => onAdd && onAdd(label)}
+                onMouseOver={() => setActiveOptionIndex(0)}
+                className={activeClass}
+                id={addingButtonId}
+                role="option"
+              >
+                <AddIcon
+                  name="fa-plus-circle"
+                  size={14}
+                  color={theme.palette.TEXT_LINK}
+                  $theme={theme}
+                />
+                <AddText themes={theme}>「{label}」を追加</AddText>
+              </AddButton>
+            )
+          }
+          return (
+            <SelectButton
+              key={`item-${label}`}
+              type="button"
+              themes={theme}
+              disabled={disabled}
+              onClick={() => onSelect(option)}
+              onMouseOver={() => setActiveOptionIndex(i)}
+              className={activeClass}
+              id={getOptionId(option)}
+              role="option"
+            >
+              {label}
+            </SelectButton>
+          )
+        })}
+
+        {isDuplicated && (
+          <NoItems themes={theme} aria-live="polite">
+            重複する選択肢は追加できません
+          </NoItems>
+        )}
+
+        {hasNoMatch && (
+          <NoItems themes={theme} aria-live="polite">
+            一致する選択肢がありません
+          </NoItems>
+        )}
+      </Container>,
+      portalRoot,
+    )
+  }
+  return {
+    renderListbox,
+    setDropdownStyle,
+    resetActiveOptionIndex: () => setActiveOptionIndex(null),
+    handleInputKeyDown,
+    listboxRef,
+    aria: {
+      listboxId,
+      activeDescendant,
+    },
+  }
+}
+
+const Container = styled.div<{ top: number; left: number; width: number; themes: Theme }>(
+  ({ top, left, width, themes }) => {
+    const { size, frame } = themes
+    return css`
+      position: absolute;
+      top: ${top}px;
+      left: ${left}px;
+      overflow-y: auto;
+      max-height: 300px;
+      width: ${width}px;
+      padding: ${size.pxToRem(size.space.XXS)} 0;
+      border-radius: ${frame.border.radius.m};
+      box-shadow: rgba(51, 51, 51, 0.3) 0 4px 10px 0;
+      background-color: #fff;
+      white-space: nowrap;
+      box-sizing: border-box;
+      &.hidden {
```

Comment:
`aria-hidden` 使うなら

```suggestion
      &[aria-hidden="true"] {
```

となるかと